[Term Entry] General SDLC: Waterfall (#1855)
Codecademy/docs: content/general/concepts/software-development-life-cycle/terms/waterfall/waterfall.md

---
Title: 'Waterfall'

> Description: 'Models a Software Development Life Cycle (SDLC) where the goal is to create a minimum viable product (MVP).'

Reviewer: ```suggestion
Description: 'A model of software development where the goal is to create a minimum viable product.'
```

> Description: 'Models a Software Development Life Cycle (SDLC) where the goal is to create a minimum viable product (MVP). Each phase in the SDLC must be completed before moving on to the next phase.'

Reviewer 2: ```suggestion
Description: 'Models a Software Development Life Cycle (SDLC) where the goal is to create a minimum viable product (MVP).'
```
Extra info doesn't need to be in the description.

Author: @SSwiniarski thanks for reviewing! I took out the extra info.

Subjects:
  - 'Mobile Development'
  - 'Web Development'
Tags:
  - 'Development'
  - 'Models'
CatalogContent:
  - 'learn-how-to-code'
  - 'paths/full-stack-engineer-career-path'
---

> The **Waterfall** Model Software Development Life Cycle (SDLC) aims to create a minimum viable product (MVP). Each phase in the SDLC must be completed before moving on to the next. It is suited to projects where features are easily defined from the start.

Reviewer: ```suggestion
The **waterfall** model is a Software Development Life Cycle (SDLC) that aims to create a minimum viable product (MVP). Each phase in this SDLC must be completed before moving on to the next. It is suited to projects where features are easily defined from the start.
```

> The Waterfall Model was the first SDLC methodology to be created and was first documented in 1970 by an American computer scientist called Dr. Winston W. Royce, though the term "Waterfall" wasn't used until later.

Reviewer: ```suggestion
The waterfall model was the first SDLC methodology to be created and was first documented in 1970 by American computer scientist Dr. Winston W. Royce, though the term "waterfall" wasn't used until later.
```

> The diagram below shows how the workflow represents the natural flow of a waterfall, with the stages being completed in linear, sequential steps:

Reviewer: ```suggestion
The diagram below shows how the workflow mirrors the natural flow of a waterfall, with the stages being completed in linear, sequential steps:
```
